packages/ketcher-react/src/script/ui/views/modal/ModalContainer.ts

```diff
@@ -14,34 +14,41 @@
  * limitations under the License.
  ***************************************************************************/
 
-import React from 'react'
 import { omit } from 'lodash/fp'
```

Comment:
please change file name to `Modal.container.ts`

Reply:
4adbc442321753f45399777f80bed9402fe16d57